app/controllers/v1/reservations_controller.rb

```diff
@@ -0,0 +1,68 @@
+class V1::ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[show edit update destroy]
```

Comment:
- [ ] Don't forget that if you are not using `show` (empty) you should remove it from the list.